src/main/java/org/jitsi_modified/service/neomedia/rtp/RemoteBitrateEstimator.java

```diff
@@ -75,12 +75,13 @@
     /**
      * Notifies this instance of an incoming packet.
      *
+     * @param nowMs the current time when this method is called
      * @param arrivalTimeMs the arrival time of the packet in millis.
      * @param timestamp the 32bit send timestamp of the packet. Note that the
      * specific format depends on the specific implementation.
      * @param payloadSize the payload size of the packet.
      * @param ssrc the SSRC of the packet.
      */
     void incomingPacketInfo(
-        long arrivalTimeMs, long timestamp, int payloadSize, long ssrc);
+        long nowMs, long arrivalTimeMs, long timestamp, int payloadSize, long ssrc);
```

Comment:
would the plan be to change the remaining `long nowMs` definitions to `Instant`s later?

Reply:
So far I haven't `Instant`-ified the Java code, but yes, eventually.